captcha/image.py

```diff
@@ -174,7 +174,8 @@ def _draw_character(c):
             try:
                 _, _, w, h = draw.textbbox((1, 1), c, font=font)
             except AttributeError:
-                w, h = draw.textsize(c, font=font)
+                w, h = draw.textbbox((1, 1), c, font=font)[2:]
```

Comment:
This is the same logic in the `try` block.

Reply:
Yea i see, But like in what cases the try block may not work? It solves the compilation error ateast.